mne/viz/_brain/_brain.py

```diff
@@ -1393,18 +1400,19 @@ def _update_glyphs(self, hemi, vectors):
                     opacity=vector_alpha,
                     name=str(hemi) + "_glyph"
                 )
-                hemi_data['glyph_dataset'] = glyph_dataset
+                hemi_data['glyph_dataset'].append(glyph_dataset)
```

Comment:
This will create N datasets/meshes for the N views, then N actors. I thought the plan was to go with 1 dataset/mesh and N actors?

Reply:
I forgot about that